SPARCbot.py

```diff
@@ -53,6 +55,20 @@ async def asyncify(fun):
 def time_format(dt):
     return dt.isoformat() + 'Z'
 
+async def get_nick_from_id(ctx,id):
+    try:
+        if not ctx.guild:
+            await timed_send(ctx, config['error_messages']['no_DM'])
+            return None
+        member = ctx.guild.get_member(id)
+        if member:
+            if member.nick:
+                return member.nick
+            user = await bot.fetch_user(id)
+            return user.name
+        return str(id)
+    except Exception:
+        raise Exception('Nickname not found')
```

Comment:
can we be more specific about which Exception we are catching and raising here, and also should this be reported as a bot message?